common/CMakeLists.txt

```diff
@@ -171,9 +171,15 @@ set(kissfft_srcs
   src/fft/kiss_fftr.c
 )
 
+set(executor_incs
+  include/pcl/experimental/execution/type_trait.hpp
+  include/pcl/experimental/execution/property.hpp
+  include/pcl/experimental/execution/executor.hpp
```

Comment:
In PCL you don't add hpp files, before the corresponding .h exists. These should be renamed to .h